src/Libraries/CoreNodeModelsWpf/Controls/ElementSelectionControl.xaml

```diff
@@ -7,7 +7,75 @@
              mc:Ignorable="d" 
              d:DesignHeight="100" d:DesignWidth="200">
     <UserControl.Resources>
-        <converters:SelectionButtonContentConverter x:Key="SelectionButtonContentConverter" />
+        <ResourceDictionary>
```

Comment:
@jesusalvino @QilongTang please see this comment I made on Long's recent PR - https://github.com/DynamoDS/Dynamo/pull/11958#discussion_r797938320

TLDR - I think this is an anti pattern and we should move the resource to the shared resource dictionary, not create it inside the control's dictionary or resources element.

Reply:
the relevant quote is:

> A ResourceDictionary defined inside of a [UserControl](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.usercontrol) carries a penalty. The platform creates a copy of such a ResourceDictionary for every instance of the UserControl. If you have a UserControl that is used a lot, then move the ResourceDictionary out of the UserControl and put it at the page level.

Reply:
I agree, converter could stay but the style can be put in https://github.com/DynamoDS/Dynamo/blob/master/src/Notifications/Styles/DynamoModern.xaml

Reply:
Done, thanks for the advice.